doc/user_guide/visualisation.rst

```diff
@@ -965,16 +965,16 @@ might lead to very slow plotting.
     >>>     marker = hs.markers.point(i, i, size=60)
     >>>     s.add_marker(marker, permanent=True)
 
-If you want to add a large amount of markers at the same time a list can be
-used:
+If you want to add a large amount of markers at the same time we advise
+to add them as an iterable (list, tuple, ...), which will be much faster:
 
 .. code-block:: python
 
     >>> from numpy.random import random
     >>> s = hs.signals.Signal2D(np.arange(300).reshape(3, 10, 10))
     >>> marker_list = []
```

Comment:
Hmmm, doesn't work. Since I use the last marker to update the plot. `TypeError: 'generator' object is not subscriptable`. I'll try to avoid this by calling the `_draw_animated` function at the end of the `add_marker` function instead.